daemon/ucrednet.go

```diff
@@ -137,6 +151,30 @@ func (wl *ucrednetListener) Accept() (net.Conn, error) {
 			Uid:    ucred.Uid,
 			Socket: ucon.LocalAddr().String(),
 		}
+		if unet.Socket == dirs.SnapSocket {
```

Comment:
that's not what I suggested, we don't want or can change this code because in principle the connection might live longer than the interface connection, we should leave this code alone and attach the interface info to the Request only in the actual checkers.

What we need here are helpers that allow:

remoteAddr = ucrednetAttachInterface(remoteAddr, interface)
uc, interface = ucrednetGetWithInterface(remoteAddr)

Reply:
sorry I misunderstood the suggestion, this is much simpler/safer.

thank you! updated.